src/Microsoft.Health.Dicom.Core.UnitTests/Extensions/ExtendedQueryTagEntryExtensionsTests.cs

```diff
@@ -87,10 +96,19 @@ public void GivenStandardTagAsKeyword_WhenNormalizing_ThenVRShouldBeFilled()
             Assert.Equal(normalized.Path, expectedPath);
         }
 
+        [Fact]
+
+        public void GivenInvalidTagWithoutVR_WhenNormalizing_ThenShouldNotThrowException()
+        {
+            // track bug  https://microsofthealth.visualstudio.com/Health/_workitems/edit/81015
+            ExtendedQueryTagEntry entry = CreateExtendedQueryTagEntry(path: "00111011", null, null, QueryTagLevel.Series, ExtendedQueryTagStatus.Ready);
+            entry.Normalize(ExtendedQueryTagStatus.Adding);
```

Comment:
nit: there should be an assertion the VR is unchanged as `null`

Reply:
The purpose of this unit test is for regresson: when tag is invalid and vr is null, NullPointerException is thrown, so I don't think we need to verify the VR?